backends/s3/s3_test.go

```diff
@@ -76,7 +77,7 @@ func getBackend(ctx context.Context, t *testing.T) (b *Backend) {
 
 func TestBackend(t *testing.T) {
 	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
-	t.Cleanup(cancel)
```

Comment:
Just curious, why this change?

Reply:
Hmmm, probably a misunderstanding of the role of `t.Cleanup` at the time I wrote this?